app/views/users/profile.html.erb

```diff
@@ -280,6 +280,17 @@
   ul.nav-tabs>li a{
     padding: 10px;
   }
+  #link_social_media{
+    padding: 50px;
+    transition: transform .2s; 
+    width: 100px;
+    height: 100px;
+    margin: 3 auto;
+}
+
+#link_social_media:hover {
+    transform: scale(1.5); 
+}
```

Comment:
The indentation on lines 289-293 should be moved to the right by 2 spaces to match that of lines 280-288? Do you think you could do that @Paarmita?

Reply:
@seafr yeah, i will do it